onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -445,8 +445,15 @@ class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain,
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 12, double_float, Dropout);
 class ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 12, double_double, Dropout);
 
+template <>
+KernelCreateInfo BuildKernelCreateInfo<void>() {
```

Comment:
Do you've have to do this for other providers as well?

Reply:
Added the function into cuda provider.